packages/cli/src/commands/styles/utils/figma.utils.ts

```diff
@@ -69,11 +69,11 @@ const isNodeFrame = (o: unknown): o is NodeFrame => {
   );
 };
 
-type ColorStyle = {
+export type ColorStyle = {
```

Comment:
Why did you export all these types? Are they used in another file?

Reply:
I presume most / many of these types would show up in (or have the potential to show up in) the Figma parser file, itself, rather than just the util file.